src/services/recim/participant.ts

```diff
@@ -8,6 +8,7 @@ export type Participant = {
   Role: string;
   Status: string;
   Notification: ParticipantNotification[];
+  IsAdmin: boolean;
```

Comment:
What is this "isAdmin" value here for? How is it to be used? If it is for authorization of global RecIM admins, it should be received from the useAuthGroups hook rather than as a part of this object.

Reply:
this is a local type for rec-im staff. the AuthGroup will be used for RecIM Directors

Reply:
To clarify, "rec-im staff" means student workers who are working for RecIM, as opposed "RecIM Directors" being full-time staff members who oversee RecIM, right?

It might be worth having a discussion (over Teams probably) about the differences in these permission levels and the pros and cons of using Auth Groups as opposed to a flag in the database.

Reply:
Yes, staff implies student workers

Reply:
We can have that discussion with ISG on our upcoming meeting. I think our MVP will be due when we get back as there is a hefty chunk of work yet to be done.